### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by mikeebowen**
Update examples to use local references and work with updated API.

### CODE REVIEW

File: Open-XML-SDK.sln
Comment (twsouthwick) [inline Open-XML-SDK.sln:62]: `doctaskssample` has abbreviations. Can you rename it to `DocumentTasksSample` using PascalCase like the other samples

File: samples/AnimatedModel3DExample/AddAnimatedModel3D.csproj
Comment (twsouthwick) [inline samples/AnimatedModel3DExample/AddAnimatedModel3D.csproj:10]: This won't work for all the samples. You shouldn't need it anyway

File: samples/DocumentTaskExample/Program.cs
Comment (twsouthwick) [inline samples/DocumentTaskExample/Program.cs:21]: nit: readonly

File: samples/DocumentTaskExample/Program.cs
Comment (twsouthwick) [inline samples/DocumentTaskExample/Program.cs:19]: can you separate classes into separate files? Nested classes can get confusing and these files are already so large

File: samples/DocumentTaskExample/Program.cs
Comment (twsouthwick) [inline samples/DocumentTaskExample/Program.cs:36]: Is there a reason not to use auto-properties?

File: samples/DocumentTaskExample/Program.cs
Comment (twsouthwick) [inline samples/DocumentTaskExample/Program.cs:70]: need a `using` here

File: samples/DocumentTaskExample/Program.cs
Comment (twsouthwick) [inline samples/DocumentTaskExample/Program.cs:97]: I wonder if we should have a single entry point for all the samples that can dynamically grow to adjust for each sample

Comment (twsouthwick) [reply samples/DocumentTaskExample/Program.cs:97]: No need to block this PR on that though

File: samples/DocumentTaskExample/README.md
Comment (twsouthwick) [inline samples/DocumentTaskExample/README.md:9]: nit: can you put a new line in between?

File: samples/Utilities/Utilities.cs
Comment (twsouthwick) [inline samples/Utilities/Utilities.cs:4]: Instead of a Utiltites project, can you add this as a common file that is linked into each project (you can look at the pattern used by Directory.Build.props/targets in the src directory)

Comment (twsouthwick) [reply samples/Utilities/Utilities.cs:4]: No need to block this PR on that though

**Review by twsouthwick — CHANGES_REQUESTED** [review]
A few changes and recommendations for follow ups.

The style looks like it may end up failing the build. Make sure to have no warnings locally, as well as with `ProjectLoadStyle=All` per https://github.com/OfficeDev/Open-XML-SDK/blob/main/CONTRIBUTING.md

File: Open-XML-SDK.sln
Comment (mikeebowen) [reply Open-XML-SDK.sln:62]: done

File: samples/DocumentTaskExample/Program.cs
Comment (mikeebowen) [reply samples/DocumentTaskExample/Program.cs:97]: It already exists, I need to add it to this file

**Comment by mikeebowen** [comment]
Made all the requested changes and made the VS linter happy.

File: samples/SVGExample/Program.cs
Comment (twsouthwick) [inline samples/SVGExample/Program.cs:10]: the header should be at the top of file

File: samples/DocumentTaskExample/README.md
Comment (twsouthwick) [reply samples/DocumentTaskExample/README.md:10]: It's also not correct type

File: samples/SVGExample/Program.cs
Comment (mikeebowen) [reply samples/SVGExample/Program.cs:10]: I'm not sure how that happened 🤔

File: samples/DocumentTaskExample/Program.cs
Comment (ThomasBarnekow) [inline samples/DocumentTaskExample/Program.cs:21]: Do you mean `< 1`?

**Review by ThomasBarnekow — COMMENTED** [review]
Looked at the examples and provided some comments. 

It seems like some of the code was generated, right? Can't help thinking that the Linq-to-XML way of creating markup is more natural in some cases at least.

File: samples/DocumentTaskExample/Program.cs
Comment (twsouthwick) [reply samples/DocumentTaskExample/Program.cs:62]: or `string.Concat(...)`. Either one would be better than `+` :)

File: samples/DocumentTaskExample/Program.cs
Comment (mikeebowen) [reply samples/DocumentTaskExample/Program.cs]: separated

File: samples/NamedSheetView/Program.cs
Comment (mikeebowen) [reply samples/NamedSheetView/Program.cs:51]: Yes, if the name space declaration isn't added, Excel won't open the file.

File: samples/SVGExample/Program.cs
Comment (mikeebowen) [reply samples/SVGExample/Program.cs:49]: This should be checking for `docPath` and `svgPath` and be in the outer scope before `presentationDocument` is assigned. I made that change.

File: samples/NamedSheetView/Program.cs
Comment (mikeebowen) [reply samples/NamedSheetView/Program.cs:53]: `args[0]` could be null that's why I'm checking. Switched to use `nameof(xlxsPath)`

File: samples/DocumentTaskExample/Program.cs
Comment (mikeebowen) [reply samples/DocumentTaskExample/Program.cs:169]: Removed

File: samples/DocumentTaskExample/Program.cs
Comment (mikeebowen) [reply samples/DocumentTaskExample/Program.cs:125]: Yes, I am, this should have been removed with the paragraph id generator. It's been removed now.

File: samples/DocumentTaskExample/Program.cs
Comment (mikeebowen) [reply samples/DocumentTaskExample/Program.cs:84]: Switched to variable

File: samples/DocumentTaskExample/Program.cs
Comment (mikeebowen) [reply samples/DocumentTaskExample/Program.cs:67]: This is a good tip, I made this change in all the new samples.

File: samples/DocumentTaskExample/Program.cs
Comment (mikeebowen) [reply samples/DocumentTaskExample/Program.cs:62]: `string.Concat(...)` is the winner 😀

File: samples/DocumentTaskExample/Program.cs
Comment (mikeebowen) [reply samples/DocumentTaskExample/Program.cs:44]: You're right, there is no need for a `StringBuilder` here. It's been changed to match your recommendation.

File: samples/DocumentTaskExample/Program.cs
Comment (mikeebowen) [reply samples/DocumentTaskExample/Program.cs:21]: Yes...

File: samples/DocumentTaskExample/README.md
Comment (mikeebowen) [reply samples/DocumentTaskExample/README.md:10]: done

**Comment by twsouthwick** [comment]
@ThomasBarnekow Agreed - it's also less for the JIT to deal with. I'm liking the direction that the XLinq stuff goes as a more lightweight way of building documents. As it matures more it'll be interesting to get feedback on performance/onboarding/maintainability/etc of that pattern.

**Comment by twsouthwick** [comment]
@mikeebowen there's a conflict you'll need to deal with

**Comment by mikeebowen** [comment]
@twsouthwick Please check to make sure I fixed the conflict correctly

**Comment by ThomasBarnekow** [comment]
@twsouthwick and @mikeebowen, would you be interested in adding an additional example (now or later) that does the same thing as an existing example (which uses the strongly typed classes) but uses the Linq-to-XML way of creating Open XML markup? I'd be happy to provide that.

File: samples/DocumentTaskExample/Program.cs
Comment (ThomasBarnekow) [reply samples/DocumentTaskExample/Program.cs:162]: @mikeebowen, you said "Removed" but the `TextId` property is still set. If you set that property, you must also set the `ParagraphId` property. It would make the example easier to understand if you did not set `TextId`.

**Comment by twsouthwick** [comment]
@ThomasBarnekow I think that would be cool. Once you feel like the shape is right with it, it would be a great chance to see a comparison.

**Comment by twsouthwick** [comment]
@mikeebowen looks like you didn't get the merge right (there's still git conflict markers, i.e. `<<<<HEAD` and such)

**Review by twsouthwick — DISMISSED** [review]
@mikeebowen This looks good enough for now. Let's keep playing with the structure of the samples and we can probably consolidate some of this.

**Review by twsouthwick — APPROVED** [review]

File: samples/common/ExampleUtilities.cs
Comment (twsouthwick) [inline samples/common/ExampleUtilities.cs:3]: need a newline

**Review by twsouthwick — APPROVED** [review]

File: samples/common/ExampleUtilities.cs
Comment (mikeebowen) [reply samples/common/ExampleUtilities.cs:4]: done

**Pull request merged**
